chart/openfaas/values.yaml

```diff
@@ -146,24 +146,30 @@ gateway:
     successThreshold: 1
 
 faasnetesPro:
-  image: ghcr.io/openfaasltd/faas-netes:0.2.9
+  image: ghcr.io/openfaasltd/faas-netes:0.3.0
```

Comment:
We have 0.3.1 now which includes the typo fix

Reply:
Bumped the image version to 0.3.1